### ISSUE TITLE
[BUG] Status line - Python Exception - IndexError: list index out of range

### ISSUE THREAD

**Issue by nivit** [coderholic/pyradio#224]
**Describe the bug**

When I tune `pyradio` to this station

Radio Art - Solo piano,http://live.radioart.com:80/fSolo_piano.mp3?dlid=

I get this 

```
Exception in thread Thread-2
(most recent call last):
File "/usr/local/lib/python3.9/threading.py", line 980, in _bootstrap_inner
self.run()
File "/usr/local/lib/python3.9/threading.py", line 917, in run
self._target(*self._args, **self._kwargs)
File "/usr/local/lib/python3.9/site-packages/pyradio/player.py", line 1276, in updateMPVStatus
if self._get_mpv_metadata(n, stop, enable_crash_detection_function):
File "/usr/local/lib/python3.9/site-packages/pyradio/player.py", line 1667, in _get_mpv_metadata
artist = a_data.split(b'"artist":"')[1].split(b'"}')[0].split(b'","')[0]                                                                │
IndexError: list index out of range 
```

and the status line doesn't show "Title: _Artist_ - _Title_" information as usual.

**System:**
 - **OS**: FreeBSD
 - **Python** version: 3.9
 - **PyRadio** Version: 0.9.2.22 (but also with 0.9.2.21) installed via FreeBSD ports
 - **Player**: `mpv` version 0.37.1

**Comment by s-n-g** [comment]
On it!

**Comment by s-n-g** [comment]
Fixed in **0.9.2.23**
Thank you for reporting it!
